lib/src/js/analyze.js

```diff
@@ -43,7 +43,7 @@ function getBehaviors(results, filePath) {
     if (behavior.contentHref != filePath) continue;
 
     behaviors.push({
-      name: behavior.is,
+      name: behavior.is.replace('Polymer.', ''),
```

Comment:
Ah ok, does it relate to the other changes though? or is it just a separate fix?